lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -157,33 +244,33 @@
          "type": "string",
          "defaultValue": "0",
          "metadata": {
-            "description": "Uninstall the MSI: Set this to 1 to uninstall the MSI. The MSI used to uninstall is the last one that was installed. It is called c:\\lansa\\MyApp.msi"
+            "description": "Uninstall the MSI: Set this to 1 to uninstall the MSI. The MSI used to uninstall, is the last one that was installed. It is called c:\\lansa\\MyApp.msi"
          }
       },
       "triggerWebConfig": {
          "type": "string",
          "defaultValue": "0",
          "metadata": {
-            "description": "Update Stack: Set this to 1  to update the web configuration"
+            "description": "Update Stack: Set this to 1 to update the web configuration"
          }
       },
       "imageOffer": {
          "type": "string",
          "defaultValue": "lansa-scalable-license",
          "allowedValues": [
             "lansa-scalable-license",
-            "visuallansa"
+            "lansa-scalable-license-preview"
```

Comment:
should this be the default?

Reply:
The default is correct. lansa-scalable-license-preview is only available as part of testing the publishing of the Azure Marketplace image .